common/v2/features/Settings/components/GeneralSettings.tsx

```diff
@@ -94,31 +102,48 @@ export default class GeneralSettings extends React.Component<SettingsProps> {
     }
   };
 
+  public renderTooltipContent = () => <div>Test</div>;
+
   public render() {
     const { globalSettings } = this.props;
     return (
-      <DashboardPanel heading="General Settings">
+      <DashboardPanel heading={translateRaw('SETTINGS_GENERAL_LABEL')}>
         <Divider />
         <SettingsField>
-          <SettingsLabel>Account Settings</SettingsLabel>
+          <SettingsLabel>
+            {translateRaw('SETTINGS_HANDLING_LABEL')}
+            <Tooltip tooltip={<span>{translateRaw('SETTINGS_TOOLTIP')}</span>}>
+              {props => (
+                <div {...props}>
+                  <SettingsTooltipIcon icon="shape" />
+                </div>
+              )}
+            </Tooltip>
+          </SettingsLabel>
           <SettingsControl>
             <Link to="/settings/import">
-              <SettingsButton secondary={true}>Import</SettingsButton>
+              <SettingsButton secondary={true}>
+                {translateRaw('SETTINGS_IMPORT_LABEL')}
+              </SettingsButton>
             </Link>
             <Link to="/settings/export">
```

Comment:
Since we are touching this file, let's update the link as well to use the centralized router like in `v2/features/Layout/Header/constants`
This will help us avoid chasing broking links in the future.